src/components/PokemonForm.js

```diff
@@ -1,35 +1,82 @@
-import React from "react";
-import { Form } from "semantic-ui-react";
+import React, { useState } from "react"
+import { Form } from "semantic-ui-react"
+import { createPokemon } from "../services/fetchers"
+
+const initializeForm = {
```

Comment:
`initialialValues` is a pretty common naming convention for default form values. 
```suggestion
const initialValues = {
```